test/parallel/test-url.js

```diff
@@ -1200,6 +1204,17 @@ var formatTests = {
     pathname: '/'
   },
 
+  // more than 255 characters in hostname which excesses the limit
+  [excessHostnameUrl]: {
```

Comment:
Nit: Maybe get rid of `excessHostnameUrl` const altogether and change this line to:

```js
[`http://${'a'.repeat(255)}.com/node`]: {
```